doc/source/conf.py

```diff
@@ -61,7 +60,7 @@
 # built documents.
 #
 # get version using setuptools-scm
-release = get_distribution("pydecorate").version
+release = get_version("pydecorate")
```

Comment:
I'm tempted to say maybe this should just be `import pydecorate; pydecorate.__version__`, but eh let's see how using this goes. I've never used it here.